gatsby-config.js

```diff
@@ -23,12 +23,12 @@ module.exports = {
         icon: 'src/assets/tnff_red.png'
       }
     },
-    {
-      resolve: 'gatsby-plugin-typography',
-      options: {
-        pathToConfigModule: 'src/utils/typography',
-      }
-    },
+    // {
```

Comment:
same thing as below, if we don't want to use it anymore feel free to remove it all together